package.json

```diff
@@ -28,12 +29,13 @@
   "author": "Microsoft Corporation",
   "license": "MIT",
   "dependencies": {
-    "@azure/communication-chat": "^1.5.4",
+    "@azure/communication-chat": "1.6.0-beta.5",
```

Comment:
Did we mean to pin to specific version this time?

<img width="319" alt="image" src="https://github.com/user-attachments/assets/284168c9-8838-4f24-92bc-ea156fdbf696" />